cmdtests/components_test.go

```diff
@@ -26,8 +26,8 @@ func TestComponentsTestSuite(t *testing.T) {
 func (s *ComponentsTestSuite) TestListStopped() {
 	require := s.Require()
 
-	out, err := s.RunCommand(context.TODO(), "components", "list")
-	require.NoError(err, out.String())
+	r := s.RunCommand("components", "list")
+	require.NoError(r.Error, r.Combined())
```

Comment:
it will be printed. Combined to see all the log to understand what happened.

Reply:
just trying to avoid huge traces like the lookout ones when a test fails and the log is outputted. I'd say that only error message should be enough, but ok.

also, what about the second?
>_also, if there was no error shouldn't we assert that `r.Err` is empty?_

Reply:
I don't understand your question. This line you comment on checks that error is empty.

Reply:
I misread `icmd API`; I meant `r.Stderr()`.
If no error is produced, and `LOG_LEVEL` is `error`, then `stderr` should be empty.

Reply:
... ouch, I realized that logs can only be configured to be verbose, or not; so log level can not be configured yet.
&rarr; so you can disregard this questions.